cmd/determinize-prow-config/main.go

```diff
@@ -0,0 +1,83 @@
+// determinize-prow-config reads and writes Prow configuration
+// to enforce formatting on the files
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/sirupsen/logrus"
+	prowconfig "k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/plugins"
+	"sigs.k8s.io/yaml"
+)
+
+type options struct {
+	prowConfigDir string
+}
+
+func (o *options) Validate() error {
+	if o.prowConfigDir == "" {
+		return errors.New("--prow-config-dir is required")
+	}
+	return nil
+}
+
+func gatherOptions() options {
+	o := options{}
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	fs.StringVar(&o.prowConfigDir, "prow-config-dir", "", "Path to the Prow configuration directory.")
+	if err := fs.Parse(os.Args[1:]); err != nil {
+		logrus.WithError(err).Fatal("could not parse input")
```

Comment:
we don't want to continue if that happens, right?

Reply:
`Fatal` will kill it, yes